packages/ai/src/Button.ts

```diff
@@ -20,18 +21,21 @@ import ButtonCss from "./generated/themes/Button.css.js";
  *
  * ### Overview
  *
- * The `ui5-ai-button` component represents a button used in AI-related scenarios.
- * It enables users to trigger actions by clicking or tapping the `ui5-ai-button`, or by pressing
- * certain keyboard keys, such as Enter.
+ * The `ui5-ai-button` component serves as a button for AI-related scenarios. Users can trigger actions by clicking or tapping the `ui5-ai-button`
+ * or by pressing keyboard keys like [Enter] or [Space].
  *
  * ### Usage
  *
- * For the `ui5-ai-button` UI, you can define one or more states of the button by placing `ai-button-state` components in its default slot.
- * Each state have a name that identifies it and can have text, icon and end icon defined (in any combination) depending on the state purpose.
+ * For the `ui5-ai-button` user interface, you can define one or more button states by placing `ui5-ai-button-state` components in their default slot.
+ * Each state has a name for identification and can include text, an icon, and an end icon, as needed for its purpose.
+ * You can define a split mode for the `ui5-ai-button`, which will results in displaying an arrow button for additional actions.
  *
- * You can choose from a set of predefined designs (the same as for regular `ui5-button` component) that allow different styling to correspond to the triggered action.
+ * You can choose from a set of predefined designs for `ui5-ai-button` (as in `ui5-button`) to match the desired styling.
  *
- * `ui5-ai-button` can be activated by clicking or tapping it. The state can be changed in `click` event handler.
+ * The `ui5-ai-button` can be activated by clicking or tapping it. You can change the button state in the click event handler. When the button is
+ * in split mode, you can activate the default button action by clicking or tapping it, or by pressing keyboard keys like [Enter] or [Space].
+ * You can activate the arrow button by clicking or tapping it, or by pressing keyboard keys like [Arrow Up], [Arrow Down], or [F4].
+ * To display additional actions, you can attach a menu to the arrow button.
```

Comment:
Please consider changing for accessibility purpose:

The `ui5-ai-button` can be activated by choosing it. You can change the button state in the click event handler. When the button is in split mode, you can activate the default button action by choosing it or by pressing keyboard keys like [Enter] or [Space]. You can activate the arrow button by choosing it or by pressing keyboard keys like [Arrow Up], [Arrow Down], or [F4]. To display additional actions, you can attach a menu to the arrow button.